.github/workflows/enos-run-k8s.yml

```diff
@@ -0,0 +1,115 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          # the Terraform wrapper will break Terraform execution in Enos because
+          # it changes the output to text when we expect it to be JSON.
+          terraform_wrapper: false
+      - name: Set up Enos
+        uses: hashicorp/action-setup-enos@v1
+        with:
+          github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+      - name: Download Docker Image
+        id: download
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.artifact-name }}
+          path: ./enos/support/downloads
+      - name: Prepare for scenario execution
+        env:
+          IS_ENT: ${{ startsWith(env.ARTIFACT_NAME, 'vault-enterprise' ) }}
+        run: |
+          mkdir -p ./enos/support/terraform-plugin-cache
+          if ${IS_ENT} == true; then
+            echo "${{ secrets.VAULT_LICENSE }}" > ./enos/support/vault.hclic || true
+            echo "edition=ent" >> $GITHUB_ENV
+            echo "edition set to ent"
+          else
+            echo "edition=oss" >> $GITHUB_ENV
+            echo "edition set to oss"
+          fi
+      - name: Run Enos scenario
+        id: run
+        # Continue once and retry to handle occasional blips when creating
+        # infrastructure.
+        continue-on-error: true
+        env:
+          ENOS_VAR_tfc_api_token: ${{ secrets.TF_API_TOKEN }}
+          ENOS_VAR_terraform_plugin_cache_dir: ../support/terraform-plugin-cache
+          ENOS_VAR_vault_build_date: ${{ env.ARTIFACT_BUILD_DATE }}
+          ENOS_VAR_vault_product_version: ${{ env.ARTIFACT_VERSION }}
+          ENOS_VAR_vault_product_revision: ${{ env.ARTIFACT_REVISION }}
+          ENOS_VAR_vault_docker_image_archive: ${{steps.download.outputs.download-path}}/${{ env.ARTIFACT_NAME }}
+          ENOS_VAR_vault_image_repository: hashicorp/vault
+        run: |
+          enos scenario run --timeout 10m0s --chdir ./enos/k8s edition:${{ env.edition }}
+      - name: Retry Enos scenario
+        id: run_retry
+        if: steps.run.outcome == 'failure'
+        env:
+          ENOS_VAR_tfc_api_token: ${{ secrets.TF_API_TOKEN }}
+          ENOS_VAR_terraform_plugin_cache_dir: ../support/terraform-plugin-cache
+          ENOS_VAR_vault_build_date: ${{ env.ARTIFACT_BUILD_DATE }}
+          ENOS_VAR_vault_product_version: ${{ env.ARTIFACT_VERSION }}
+          ENOS_VAR_vault_product_revision: ${{ env.ARTIFACT_REVISION }}
+          ENOS_VAR_vault_docker_image_archive: ${{steps.download.outputs.download-path}}/${{ env.ARTIFACT_NAME }}
+          ENOS_VAR_vault_image_repository: hashicorp/vault
+        run: |
+          enos scenario run --timeout 10m0s --chdir ./enos/k8s edition:${{ env.edition }}
+      - name: Destroy Enos scenario
+        if: ${{ always() }}
+        # With Enos version 0.0.11 the destroy step returns an error if the infrastructure
```

Comment:
We released the fix for this, but might be worth keeping around just in case?

Reply:
Right, thanks for that. I'll remove this.